Learn/Managers/DataManager.swift

```diff
@@ -21,11 +21,11 @@ final class DataManager {
     let settings: LoopSettings
 
     init(
-        basalRateSchedule: BasalRateSchedule? = UserDefaults.appGroup?.basalRateSchedule,
-        carbRatioSchedule: CarbRatioSchedule? = UserDefaults.appGroup?.carbRatioSchedule,
-        defaultRapidActingModel: ExponentialInsulinModelPreset? = UserDefaults.appGroup?.defaultRapidActingModel,
-        insulinSensitivitySchedule: InsulinSensitivitySchedule? = UserDefaults.appGroup?.insulinSensitivitySchedule,
-        settings: LoopSettings = UserDefaults.appGroup?.loopSettings ?? LoopSettings()
+        basalRateSchedule: BasalRateSchedule? = UserDefaults.appGroup?.legacyBasalRateSchedule,
+        carbRatioSchedule: CarbRatioSchedule? = UserDefaults.appGroup?.legacyCarbRatioSchedule,
+        defaultRapidActingModel: ExponentialInsulinModelPreset? = UserDefaults.appGroup?.legacyDefaultRapidActingModel,
+        insulinSensitivitySchedule: InsulinSensitivitySchedule? = UserDefaults.appGroup?.legacyInsulinSensitivitySchedule,
+        settings: LoopSettings = UserDefaults.appGroup?.legacyLoopSettings ?? LoopSettings()
```

Comment:
I assume for `Learn` using the legacy is ok?

Reply:
Yes, Lean needs a lot of updating; using StoredSettings will have to wait.